package.json

```diff
@@ -30,12 +30,15 @@
     "update": "nx migrate latest",
     "workspace-schematic": "nx workspace-schematic",
     "dep-graph": "nx dep-graph",
-    "help": "nx help"
+    "help": "nx help",
+    "firebase-api:emulators-start-firestore": "nx run firebase-api:emulators-start-firestore",
+    "firebase-api:test-firestore": "nx run firebase-api:test-firestore"
```

Comment:
Delete this. We already have a test function you can call with `yarn test firestore-api`. Use this, or modify it to suit you needs.